lib/routes/splash_page.dart

```diff
@@ -46,9 +48,19 @@ class SplashPageState extends State<SplashPage> {
   Widget build(BuildContext context) {
     return Scaffold(
         body: Stack(children: <Widget>[
-      Center(
-        child: SizedBox(),
-      ),
-    ]));
+          Center(
+            child: SizedBox(),
+          ),
+        ]));
+    // return   FutureBuilder<FirebaseRemoteConfig>(
```

Comment:
did you mean to leave this here?

Reply:
also a single line on lib/routes/home/qr_action_button.dart

Reply:
The commented code was removed.